<ui version="4.0" >
 <class>VideoFetcher</class>
 <widget class="QWidget" name="VideoFetcher" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>396</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="labelInfo" >
     <property name="text" >
      <string>Enter links to flash video webpages (only &lt;a href="showServices">certain services&lt;/a> are supported!):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QTextEdit" name="textLinks" >
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Decoded links:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QTextEdit" name="textDecoded" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QPushButton" name="pushDecode" >
     <property name="text" >
      <string>Decode</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="pushDownload" >
     <property name="text" >
      <string>Download decoded</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>340</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
